### PR TITLE
chore: Update Logger imports

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes

Updated all imports to point to logger wrapper util in the ui package, that way regardless of what logging solution we use in the future we have to make changes in only one place.

#### Description of how you validated changes

Built packages locally

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/src/helpers/logger/logger.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  | 'StorageManager';
+
+export enum LOG_LEVEL {
+  DEBUG = 'DEBUG',
+  ERROR = 'ERROR',
+  INFO = 'INFO',
+  WARN = 'WARN',
+}
+
+class Logger {
```
Comment (ioanabrooks) [inline packages/ui/src/helpers/logger/logger.ts:18]: Simplistic version of a Logger to cover our existing usecases, open to add more if needed.

**Comment by thaddmt** [comment]
Not sure we want to update this yet if all that is changing is where it is imported from. Open to discussion though

**Comment by ioanabrooks** [comment]
> Not sure we want to update this yet if all that is changing is where it is imported from. Open to discussion though

Providing our own implementation aside, having a wrapper around it is good bc if the import changes in the future, we only have to modify in one place.

File: packages/ui/src/helpers/logger/logger.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  | 'AccountSettings'
+  | 'Auth'
+  | 'Geo'
+  | 'Icon'
+  | 'Notifications'
+  | 'StateMachine'
+  | 'StorageImage'
+  | 'StorageManager';
+
+export const getLogger = (category: LoggerCategory) =>
```
Comment (reesscot) [inline packages/ui/src/helpers/logger/logger.ts:13]: nit: could be `getUILogger` to indicate that it's prefixing with `AmplifyUI:`?
```suggestion
export const getLogger = (category: LoggerCategory) =>
```

Comment (ioanabrooks) [reply packages/ui/src/helpers/logger/logger.ts:13]: Is that necessary? It's in the ui package, not seeing a future where we would have different or multiple prefixes.

Comment (reesscot) [reply packages/ui/src/helpers/logger/logger.ts:13]: Yeah, it was a loosely held nit. I'm good with getLogger

**Review by calebpollman — APPROVED** [review]

**Pull request merged**